app/src/androidTest/java/com/github/sdp/mediato/MyProfileFragmentTest.java

```diff
@@ -61,7 +67,7 @@
 
 @RunWith(AndroidJUnit4.class)
 public class MyProfileFragmentTest {
-    private final static int STANDARD_USER_TIMEOUT = 10;
+    private final static int STANDARD_USER_TIMEOUT = 60;
```

Comment:
Why is the timeout changed to 60 seconds ?

Reply:
Changed it for my slow laptop but will change it back.